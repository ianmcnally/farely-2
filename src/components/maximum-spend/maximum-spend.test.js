import React from 'react'
import renderShallow from 'render-shallow'
import MaximumSpend from './maximum-spend'

describe('with required props', () => {
  let component

  beforeAll(() => {
    component = renderShallow(<MaximumSpend />).output
  })

  it('renders the input', () => {
    expect(component).toMatchSnapshot()
  })

})
