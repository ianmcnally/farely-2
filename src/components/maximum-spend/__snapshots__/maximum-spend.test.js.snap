// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`with props.onChange renders the input with an onChange 1`] = `
<p
  className="mb0"
>
  <label
    className="db fw6 lh-copy f5"
    htmlFor="maximum-spend"
  >
    Max to spend
  </label>
  <CurrencyInput
    className="pa2 w-100 input-reset ba measure b--black-20"
    id="maximum-spend"
    onChange={[Function]}
    type="number"
  />
</p>
`;

exports[`with props.value renders the input with a value 1`] = `
<p
  className="mb0"
>
  <label
    className="db fw6 lh-copy f5"
    htmlFor="maximum-spend"
  >
    Max to spend
  </label>
  <CurrencyInput
    className="pa2 w-100 input-reset ba measure b--black-20"
    id="maximum-spend"
    type="number"
    value="hey"
  />
</p>
`;

exports[`with required props renders the input 1`] = `
<p
  className="mb0"
>
  <label
    className="db fw6 lh-copy f5"
    htmlFor="maximum-spend"
  >
    Max to spend
  </label>
  <CurrencyInput
    className="pa2 w-100 input-reset ba measure b--black-20"
    id="maximum-spend"
    type="number"
  />
</p>
`;
